test/integration/concierge_impersonation_proxy_test.go

```diff
@@ -1776,6 +1778,84 @@ func TestImpersonationProxy(t *testing.T) { //nolint:gocyclo // yeah, it's compl
 		)
 	})
 
+	t.Run("using externally provided TLS serving cert", func(t *testing.T) {
+		var externallyProvidedCA *certauthority.CA
+		externallyProvidedCA, err = certauthority.New("Impersonation Proxy Integration Test CA", 1*time.Hour)
+		require.NoError(t, err)
+
+		var externallyProvidedTLSServingCertPEM, externallyProvidedTLSServingKeyPEM []byte
+		externallyProvidedTLSServingCertPEM, externallyProvidedTLSServingKeyPEM, err = externallyProvidedCA.IssueServerCertPEM([]string{proxyServiceEndpoint}, nil, 1*time.Hour)
+		require.NoError(t, err)
+
+		externallyProvidedTLSServingCertSecretName := "external-tls-cert-secret-name" //nolint:gosec // this is not a credential
+		externallyProvidedTLSServingCertSecret := &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      externallyProvidedTLSServingCertSecretName,
+				Namespace: env.ConciergeNamespace,
+			},
+			Type: corev1.SecretTypeTLS,
+			Data: map[string][]byte{
+				v1.TLSCertKey:       externallyProvidedTLSServingCertPEM,
```

Comment:
Could also have an integration test (new or addition to same) which tests that setting `ca.crt` in the Secret also works as an alternative to setting the `CredentialIssuer.spec.tls.certificateAuthorityData`?

Reply:
Did we discuss this on zoom the other day?

Reply:
Yeah.